diff --git a/node_modules/vue/types/vue.d.ts b/node_modules/vue/types/vue.d.ts
old mode 100644
new mode 100755
index 204f9cc..31d3ac1
--- a/node_modules/vue/types/vue.d.ts
+++ b/node_modules/vue/types/vue.d.ts
@@ -24,7 +24,6 @@ export interface Vue {
   readonly $el: Element;
   readonly $options: ComponentOptions<Vue>;
   readonly $parent: Vue;
-  readonly $root: Vue;
   readonly $children: Vue[];
   readonly $refs: { [key: string]: Vue | Element | Vue[] | Element[] };
   readonly $slots: { [key: string]: VNode[] | undefined };
